Mismatched types on language enum: expected `u8`, found `i8`

This change upstream https://github.com/devkitPro/libctru/pull/553 changed the enum repr for `CFG_Language` from `u8` to `i8`. It seems this was done to accommodate the error applet which interprets the language enum differently (0 means use the system default). There is no actual change to the values of the existing enum constants, just their repr.

```
error[E0308]: mismatched types
  --> /home/-/.cargo/git/checkouts/ctru-rs-1206856f7a5d76bc/4a15ec9/ctru-rs/src/services/cfgu.rs:35:16
   |
35 |     Japanese = ctru_sys::CFG_LANGUAGE_JP,
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`

error[E0308]: mismatched types
  --> /home/-/.cargo/git/checkouts/ctru-rs-1206856f7a5d76bc/4a15ec9/ctru-rs/src/services/cfgu.rs:37:15
   |
37 |     English = ctru_sys::CFG_LANGUAGE_EN,
   |               ^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`

error[E0308]: mismatched types
  --> /home/-/.cargo/git/checkouts/ctru-rs-1206856f7a5d76bc/4a15ec9/ctru-rs/src/services/cfgu.rs:39:14
   |
39 |     French = ctru_sys::CFG_LANGUAGE_FR,
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`

```

--- a/ctru-rs/src/services/cfgu.rs
+++ b/ctru-rs/src/services/cfgu.rs
@@ -29,7 +29,7 @@ pub enum Region {
 /// Language set for the console's OS.
 #[doc(alias = "CFG_Language")]
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
-#[repr(u8)]
+#[repr(i8)]
 pub enum Language {
     /// Japanese.
     Japanese = ctru_sys::CFG_LANGUAGE_JP,
@@ -174,7 +174,7 @@ impl Cfgu {
         let mut language: u8 = 0;
 
         ResultCode(unsafe { ctru_sys::CFGU_GetSystemLanguage(&mut language) })?;
-        Ok(Language::try_from(language).unwrap())
+        Ok(Language::try_from(language as i8).unwrap())
     }
 
     /// Check if NFC is supported by the console.
@@ -244,7 +244,7 @@ impl Drop for Cfgu {
 }
 
 from_impl!(Region, u8);
-from_impl!(Language, u8);
+from_impl!(Language, i8);
 from_impl!(SystemModel, u8);
 
 impl TryFrom<u8> for Region {
@@ -264,10 +264,10 @@ impl TryFrom<u8> for Region {
     }
 }
 
-impl TryFrom<u8> for Language {
+impl TryFrom<i8> for Language {
     type Error = ();
 
-    fn try_from(value: u8) -> Result<Self, Self::Error> {
+    fn try_from(value: i8) -> Result<Self, Self::Error> {
         match value {
             ctru_sys::CFG_LANGUAGE_JP => Ok(Language::Japanese),
             ctru_sys::CFG_LANGUAGE_EN => Ok(Language::English),

